.test-defs/cmd/helper.go

```diff
@@ -43,6 +43,31 @@ func UpdateWorkerZone(shoot *gardenv1beta1.Shoot, cloudprovider gardenv1beta1.Cl
 	return nil
 }
 
+// UpdateMachineImage updates the machine image name and version of a shoot if a image is provided.
+func UpdateMachineImage(shoot *gardenv1beta1.Shoot, cloudprovider gardenv1beta1.CloudProvider, image, version string) error {
+	if image != "" {
+		machineImage := &gardenv1beta1.MachineImage{
+			Name: image,
+			Version: version,
+		}
+		switch cloudprovider {
+		case gardenv1beta1.CloudProviderAWS:
+			shoot.Spec.Cloud.AWS.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderGCP:
+			shoot.Spec.Cloud.GCP.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderAzure:
+			shoot.Spec.Cloud.Azure.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderOpenStack:
+			shoot.Spec.Cloud.OpenStack.MachineImage = machineImage
+		case gardenv1beta1.CloudProviderAlicloud:
+			shoot.Spec.Cloud.Alicloud.MachineImage = machineImage
```

Comment:
Packet is missing

Reply:
Also added packet to the other update functions

Reply:
Great